cartridges/demo_dwscript/cartridge/models/task.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+// Create a function to get product by a given ID​
+/**
+ * @param {dw.catalog.Product} product
+ * @returns {String} product ID
+ */
+function getProductID(product) {
+    return product.getID();
+}
+
+// Create a function to get product category by given product​
+/**
+ * @param {dw.catalog.Product} product
+ * @returns {Collection} product categories
+ */
+function getProductCategory(product) {
+    return product.getCategories();
+}
+
+// Create a function to get different product prices for a given product​
+/**
+ * @param {dw.catalog.Product} product
+ * @returns {ProductPriceModel} product prices
+ */
+function getProductPrice(product) {
+    return product.getPriceModel();
+}
+
+// Create a function to get catalog main categories​
+/**
+ *
+ * @param {dw.catalog.Catalog} catalog
+ * @returns {Category}  category
+ */
+function getMainCategory(catalog) {
+    return catalog.getRoot();
+    // CatalogMgr.getCatalog('catalogID').getRootCategory().getSubCategories()
+}
+
+// Create a function to get customer by ID​
+/**
+ *
+ * @param {dw.customer.Customer} customer
+ * @returns {String} customer ID
+ */
+function getCustomerID(customer) {
+    return customer.getID();
+}
+
+// Create a function to check if a given customer is assigned to a given customer group​
+/**
+ * @param {dw.customer.CustomerMgr} customer
+ * @returns {boolean} true or false
+ *
+ */
+function getCustomerGroup(customer, customerGroup) {
+    const groups = customer.getCustomerGroups();
```

Comment:
This will do the trick, but is there any other customer method you can use?

Reply:
Probably isMemberOfCustomerGroup